src/app/helptext/task-calendar/initshutdown/initshutdown.ts

```diff
@@ -4,25 +4,27 @@ import {Validators} from '@angular/forms';
 export default {
     ini_type_placeholder: T('Type'),
     ini_type_tooltip: T('Select <i>Command</i> for an executable or\
-                <i>Script</i> for an executable script.'),
+                         <i>Script</i> for an executable script.'),
```

Comment:
the text to these should be as far to the left as they possibly can with only a single space indention (otherwise the string extractor will add the extra spaces)

Reply:
Ah, ok. I didn't realize there was a change from the previous help text standard.